src/lib/components/banner.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
+  import { closedBannerId, close } from '$lib/stores/banner';
+  import { cluster } from '$lib/stores/cluster';
+
+  const severities = {
+    High: 'high',
+    Medium: 'medium',
+    Low: 'low',
+  };
+
+  $: recommended = $cluster?.versionInfo?.recommended;
```

Comment:
I'm not totally sure this is the best data structure since it's just one call that we're not doing anything with, but we can refactor it later.